sdk/identity/identity-cache-persistence/.gitignore

```diff
@@ -0,0 +1,2 @@
+src/**/*.js
```

Comment:
is something compiling the TS outside of dist?

Reply:
This was probably inherited from identity. Nothing should be compiling to JS outside of dist*.